compiler/x/codegen/X86Ops.ins

```diff
@@ -3774,6 +3774,38 @@ INSTRUCTION(VPERM2I128RegRegImm1, vperm2i128 ,
             BINARY(VEX_L256, VEX_vReg_, PREFIX_66, REX__, ESCAPE_0F3A, 0x46, 0, ModRM_RM__, Immediate_1),
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_ByteImmediate | IA32OpProp_SourceRegisterInModRM | IA32OpProp_UsesTarget),
             PROPERTY1(IA32OpProp1_XMMSource | IA32OpProp1_XMMTarget)),
+INSTRUCTION(VFMADDSS, vfmadd132ss,
```

Comment:
The names of these new instructions do not match our current naming scheme: the instruction mnemonic + operand form. For instance, `vfmadd132ss` should be "VFMADD132SSRegRegReg".

Reply:
In addition, I strongly recommend to keep the `132` in our instruction names in case of possible future need for other FMA instruction like the `213` or `231` versions.

Reply:
done